src/less/components/file/hx-file-control.less

```diff
@@ -12,7 +12,7 @@ hx-file-control {
   }
 
   > input[type="file"] {
-    #Mix.hideInput();
+    // #Mix.hideInput(); @include hxInput(visuallyHidden);
```

Comment:
https://user-images.githubusercontent.com/55139500/73269801-7e542400-4203-11ea-83bd-edd2c4d6da29.png
I have commented beacuse of this #Mix.hideInput please find details in image.

Reply:
Ok, so the mixins are never loaded by the File component LESS.  The strategy of commenting out broken code is equivalent to saying "My arm is numb, so I'm going to cut it off so that it's no longer numb."

A better approach would be to add an import to the top of the file so that the mixin can be used again.  

```less
@import 'mixins';
```

_On a related note:_
I understand why the compiler is complaining that the mixin isn't defined (because `src/less/mixins.less` is never imported). However, I'm curious as to how the compiler wasn't complaining beforehand. (Likely because the pipeline was updated to handle errors, as seen in your screenshot, so Travis no longer fails the build.)

Reply:
Changes have been done, please review.